src/courses/mappers/03.md

```diff
@@ -7,7 +7,14 @@ author: Charles Hu
 
 ## Where Is OHDF Used?
 
-OHDF is part of the SAF normalization process (see below) and is officially implemented and used in a plethora of tools and libraries including InspecJS, OHDF Converters, the SAF CLI, and Heimdall.
+The SAF has five core capabilities, which you can read more about [here](../user/03.md). Here is how OHDF relates to these five pillars:
+
+- **Plan**: The OHDF spec is written such that it includes all the information described in the guidance, as well as all the relavant metadata. Thus, the information is carried alongside the results. 
+- **Harden**: The OHDF spec is not really relevant in this case. 
+- **Validate**: InSpec's results format is a subset of the OHDF spec. This means that InSpec itself is natively supported! Moreover, it's easy to write InSpec such that all the guidance information is passed through into both the OHDF profile and OHDF results formats. 
+- **Normalize**: Normalization is the primary use of OHDF. With the use of OHDF Converters, we are able to map many different security results formats to a standard format (OHDF). These converters can either be implemented directly, or accessed via the SAF CLI. 
```

Comment:
I mean considering how much inspec we write and utilize, I would say that 'validation' is also a primary usage of OHDF.  Also Heimdall is our primary visualization thing, and it's not gonna work without HDF.  Let's rephrase this section a bit to get rid of the 'primary' stuff.

Reply:
Addressed.